.gitlab/source_test/ebpf.yml

```diff
@@ -53,6 +53,33 @@
     paths:
       - $DD_AGENT_TESTING_DIR/site-cookbooks/dd-system-probe-check/files
 
+.tests_windows_secagent:
```

Comment:
Can you add these to a different file in this directory? They have nothing to do with eBPF.

Reply:
I realize `tests_windows_sysprobe` is already in this file, but that should probably be moved too.